module_utils/network_lsr/argument_validator.py

```diff
@@ -246,6 +247,62 @@ def _validate_string_min_length(self, value):
             return True
 
 
+class ArgValidatorRouteTable(ArgValidator):
+    def __init__(
+        self,
+        name,
+        required=False,
+        default_value=None,
+    ):
+        ArgValidator.__init__(self, name, required, default_value)
+
+    def _validate_impl(self, value, name):
+        norm_value = None
```

Comment:
IMHO, it would be great to avoid unnecessary abbreviations in variable such as "norm". Since this will be the routing table, using `table` here seems like a good choice.